common/beerocks/bwl/include/bwl/ap_wlan_hal.h

```diff
@@ -325,7 +325,10 @@ class ap_wlan_hal : public virtual base_wlan_hal {
      *
      * @return true on success or false on error.
      */
-    virtual bool set_tx_power_limit(int tx_pow_limit) = 0;
+    virtual bool set_tx_power_limit(int tx_pow_limit)
```

Comment:
I think parameter should be uint32_t

Reply:
This was already like that apparently and I reverted this original change so I prefer not to change this as part of this PR.